The Unofficial Hollow Knight RPG (Beta Update #1, bugfixes and new features)

Ccmaci opened the pull request:
<!-- ATTENTION: This Pull Request template changed on 03/17/22. Please ensure that you are completing this template to the fullest extent possible. -->

# Submission Checklist
## Required

<!-- Check these off by adding an 'x' to each of these boxes. If you fail to meet all these criteria, your PR will be rejected. -->

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

## New Sheet Details

<!-- If you are submitting a new sheet to the repository, please fill in any empty spaces indicated by < >. -->

- The name of this game is: < The Unofficial Hollow Knight RPG >  _(i.e. Dungeons & Dragons 5th Edition, The Dresden Files RPG)_
- The publisher of this game is: < The HKRPG Team > _(i.e. Wizards of the Coast, Evil Hat)_
- The name of this game system/family is: < The Unofficial Hollow Knight RPG > _(i.e. Dungeons & Dragons, FATE)_

- [x] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.

<!-- Please check any that apply: -->

- [x] This sheet has been made on behalf of, or by, the game's publisher.
- [x] This game is not a traditionally published game, but a copy of the game rules can be purchased/downloaded/found at: < https://hkrpg-team.itch.io/hollow-knight-rpg >
- [x] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system.

# Changes / Description

<This is beta test update #1 for the Unofficial HK RPG I've been working on over the last 2 months. Page 1 has been fully developed, with most bugs and features I missed accounted for. Note to the Roll20 team, the name of the sheet has changed a little; please use the name listed in this Pull Request Template, not the name it currently has on Roll20.>

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/Hollow_Knight_Unofficial_TTRPG/roll20-character-sheets) *Roll20 Internal Use only.*

Ccmaci commented:
Here's the changes and fixes:

Additions:
- Durability being set to 0 for Shell Absorption or worn Armor will remove protection effects given by that armor (including additional soak rolls/rerolls) (this means you’ll need to set durability of your shell to a non-zero number for absorption to work, even if it doesn’t use durability).
- Added “+Belt” and “+Footwork” to Modifiers Page
- Added “+Parry Dice (Weapon Specific)”  and “+Parry Dice (Shield Specific)”
- Added Shield Bashes, including a working “Balanced” option on the modifiers page.
- Reroll modifiers for Soak, Cute, Spook, Might Shell Insight or Grace based rolls, and like just a whole bunch of stuff.
- Custom success threshold for Soak rolls.
- Added an “End Turn” button
- Added Status Effect counters, which automatically decrease when clicking “End Turn”.

Fixes:
- Fixed glitch where soaking with soft body would initiate a reroll if shell had a “.5”, which shouldn’t happen when soft body is active.
- Fixed “Calculated Damage” math so rollover damage cap increases if dedicated stamina of your last attack is higher than your equipped weapon’s damage.
- Removed arrows from number inputs where unnecessary, due to some computers causing overlapping of text
- Minor spacing and formatting changes
- “Attack Roll Toggle” now switched to ‘Melee’ and ‘Ranged’. Modifiers page allows you to choose what attribute is used for your Melee and Ranged attacks.
- Stamina Tax resets when clicking “End Turn”
- Paralysis SE ticks down when clicking “End Turn”
